keyboards/2_milk/keymaps/mikethetiger/rules.mk

```diff
@@ -0,0 +1,34 @@
+﻿# MCU name
+MCU = atmega32u4
+
+# Processor frequency.
```

Comment:
Lines 4 to 15, and 18 to 20 can be removed - QMK provides these as defaults for this MCU.